Date pickers not rendered with "3.0.0-rc.0"

### Describe the bug

Date pickers are not rendered when using *some* combination of the React Material renderers with Material UI.

### Expected behavior

All pickers (date, date-time, time) should work

### Steps to reproduce the issue

- Reported [here](https://jsonforms.discourse.group/t/datepicker-not-rendering/942/3).
- Reproducible [here](https://github.com/eclipsesource/jsonforms-react-seed/pull/97)

### Screenshots

_No response_

### In which browser are you experiencing the issue?

Chrome 104.0.5112.79

### Framework

React

### RendererSet

Material

### Additional context

_No response_

Weird, I added a npm package (`@mui/x-date-pickers`) to `packages/material` and run `npm run init` and `npm run build` and get the error below. Is there a specific way I need to add packages?
```
lerna ERR! npm run build exited 1 in '@jsonforms/material-renderers'
lerna ERR! npm run build stdout:

> @jsonforms/material-renderers@3.0.0-rc.0 build C:\src\jsonforms\packages\material
> rollup -c rollup.config.js


lerna ERR! npm run build stderr:

src/index.ts → lib/jsonforms-react-material.esm.js...
[!] (plugin rpt2) Error: C:/src/jsonforms/packages/material/src/layouts/MaterialCategorizationStepperLayout.tsx(117,6): semantic error TS2786: 'Hidden' cannot be used as a JSX component.
  Its element type 'ReactElement<any, any> | Component<any, any, any>' is not a valid JSX element.
    Type 'Component<any, any, any>' is not assignable to type 'Element | ElementClass'.
      Type 'Component<any, any, any>' is not assignable to type 'ElementClass'.
        The types returned by 'render()' are incompatible between these types.
          Type 'import("C:/src/jsonforms/node_modules/@types/react/index").ReactNode' is not assignable to type 'React.ReactNode'.
            Type '{}' is not assignable to type 'ReactNode'.
              Type '{}' is missing the following properties from type 'ReactPortal': key, children, type, props
src\layouts\MaterialCategorizationStepperLayout.tsx
Error: C:/src/jsonforms/packages/material/src/layouts/MaterialCategorizationStepperLayout.tsx(117,6): semantic error TS2786: 'Hidden' cannot be used as a JSX component.
  Its element type 'ReactElement<any, any> | Component<any, any, any>' is not a valid JSX element.
    Type 'Component<any, any, any>' is not assignable to type 'Element | ElementClass'.
      Type 'Component<any, any, any>' is not assignable to type 'ElementClass'.
        The types returned by 'render()' are incompatible between these types.
          Type 'import("C:/src/jsonforms/node_modules/@types/react/index").ReactNode' is not assignable to type 'React.ReactNode'.
            Type '{}' is not assignable to type 'ReactNode'.
              Type '{}' is missing the following properties from type 'ReactPortal': key, children, type, props
    at error (C:\src\jsonforms\node_modules\rollup\dist\shared\rollup.js:158:30)
    at throwPluginError (C:\src\jsonforms\node_modules\rollup\dist\shared\rollup.js:21769:12)
    at Object.error (C:\src\jsonforms\node_modules\rollup\dist\shared\rollup.js:22472:20)
    at Object.error (C:\src\jsonforms\node_modules\rollup\dist\shared\rollup.js:21945:38)
    at RollupContext.error (C:\src\jsonforms\node_modules\rollup-plugin-typescript2\src\rollupcontext.ts:37:18)
    at C:\src\jsonforms\node_modules\rollup-plugin-typescript2\src\print-diagnostics.ts:41:11
    at arrayEach (C:\src\jsonforms\node_modules\rollup-plugin-typescript2\node_modules\lodash\lodash.js:530:11)
    at Function._.each [as forEach] (C:\src\jsonforms\node_modules\rollup-plugin-typescript2\node_modules\lodash\lodash.js:9410:14)
    at printDiagnostics (C:\src\jsonforms\node_modules\rollup-plugin-typescript2\src\print-diagnostics.ts:9:2)
    at Object.transform (C:\src\jsonforms\node_modules\rollup-plugin-typescript2\src\index.ts:244:5)
```

--- a/packages/material/package.json
+++ b/packages/material/package.json
@@ -79,8 +79,8 @@
     "@jsonforms/core": "3.0.0-rc.0",
     "@jsonforms/react": "3.0.0-rc.0",
     "@mui/icons-material": "^5.0.0",
-    "@mui/lab": "^5.0.0-alpha.54",
-    "@mui/material": "^5.0.0"
+    "@mui/material": "^5.0.0",
+    "@mui/x-date-pickers": "^5.0.0-beta.5"
   },
   "devDependencies": {
     "@emotion/react": "^11.5.0",
@@ -88,8 +88,8 @@
     "@jsonforms/core": "^3.0.0-rc.0",
     "@jsonforms/react": "^3.0.0-rc.0",
     "@mui/icons-material": "^5.2.0",
-    "@mui/lab": "^5.0.0-alpha.58",
     "@mui/material": "^5.2.2",
+    "@mui/x-date-pickers": "^5.0.0-beta.5",
     "@types/enzyme": "^3.10.3",
     "@types/react-dom": "^17.0.9",
     "@wojtekmaj/enzyme-adapter-react-17": "^0.6.3",

--- a/packages/material/src/controls/MaterialDateControl.tsx
+++ b/packages/material/src/controls/MaterialDateControl.tsx
@@ -36,8 +36,8 @@ import { FormHelperText, Hidden } from '@mui/material';
 import {
   DatePicker,
   LocalizationProvider 
-} from '@mui/lab';
-import AdapterDayjs from '@mui/lab/AdapterDayjs';
+} from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import {
   createOnChangeHandler,
   getData,
@@ -96,15 +96,16 @@ export const MaterialDateControl = (props: ControlProps)=> {
         <DatePicker
           label={label}
           value={value}
-          clearable
           onChange={onChange}
           inputFormat={format}
           disableMaskedInput
           views={views}
           disabled={!enabled}
-          cancelText={appliedUiSchemaOptions.cancelLabel}
-          clearText={appliedUiSchemaOptions.clearLabel}
-          okText={appliedUiSchemaOptions.okLabel}
+          componentsProps={{
+            actionBar: {
+              actions: (variant) => (variant === 'desktop' ? [] : ['clear', 'cancel', 'accept'])
+            }
+          }}
           renderInput={params => (
             <ResettableTextField 
               {...params}

--- a/packages/material/src/controls/MaterialDateTimeControl.tsx
+++ b/packages/material/src/controls/MaterialDateTimeControl.tsx
@@ -36,8 +36,8 @@ import { FormHelperText, Hidden } from '@mui/material';
 import {
   DateTimePicker,
   LocalizationProvider 
-} from '@mui/lab';
-import AdapterDayjs from '@mui/lab/AdapterDayjs';
+} from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import {
   createOnChangeHandler,
   getData,
@@ -98,16 +98,17 @@ export const MaterialDateTimeControl = (props: ControlProps) => {
         <DateTimePicker
           label={label}
           value={value}
-          clearable
           onChange={onChange}
           inputFormat={format}
           disableMaskedInput
           ampm={!!appliedUiSchemaOptions.ampm}
           views={views}
           disabled={!enabled}
-          cancelText={appliedUiSchemaOptions.cancelLabel}
-          clearText={appliedUiSchemaOptions.clearLabel}
-          okText={appliedUiSchemaOptions.okLabel}
+          componentsProps={{
+            actionBar: {
+              actions: (variant) => (variant === 'desktop' ? [] : ['clear', 'cancel', 'accept'])
+            }
+          }}
           renderInput={params => (
             <ResettableTextField 
               {...params}
@@ -129,7 +130,8 @@ export const MaterialDateTimeControl = (props: ControlProps) => {
               onBlur={onBlur}
               variant={'standard'}
             />
-          )}
+          )
+          }
         />
         <FormHelperText error={!isValid && !showDescription}>
           {firstFormHelperText}

--- a/packages/material/src/controls/MaterialTimeControl.tsx
+++ b/packages/material/src/controls/MaterialTimeControl.tsx
@@ -36,8 +36,8 @@ import { FormHelperText, Hidden } from '@mui/material';
 import {
   TimePicker,
   LocalizationProvider 
-} from '@mui/lab';
-import AdapterDayjs from '@mui/lab/AdapterDayjs';
+} from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import {
   createOnChangeHandler,
   getData,
@@ -98,16 +98,17 @@ export const MaterialTimeControl = (props: ControlProps) => {
         <TimePicker
           label={label}
           value={value}
-          clearable
           onChange={onChange}
           inputFormat={format}
           disableMaskedInput
           ampm={!!appliedUiSchemaOptions.ampm}
           views={views}
           disabled={!enabled}
-          cancelText={appliedUiSchemaOptions.cancelLabel}
-          clearText={appliedUiSchemaOptions.clearLabel}
-          okText={appliedUiSchemaOptions.okLabel}
+          componentsProps={{
+            actionBar: {
+              actions: (variant) => (variant === 'desktop' ? [] : ['clear', 'cancel', 'accept'])
+            }
+          }}
           renderInput={params => (
             <ResettableTextField 
               {...params}

--- a/packages/material/test/renderers/MaterialDateTimeControl.test.tsx
+++ b/packages/material/test/renderers/MaterialDateTimeControl.test.tsx
@@ -209,10 +209,10 @@ describe('Material date time control', () => {
       </JsonFormsStateProvider>
     );
     const input = wrapper.find('input').first();
-    (input.getDOMNode() as HTMLInputElement).value ='1961-12-94 20:15'; 
+    (input.getDOMNode() as HTMLInputElement).value ='1961-12-12 20:15'; 
     input.simulate('change', input);
     expect(onChangeData.data.foo).toBe(
-      dayjs('1961-12-94 20:15').format()
+      dayjs('1961-12-12 20:15').format()
     );
   });
 

--- a/packages/material/test/renderers/MaterialTimeControl.test.tsx
+++ b/packages/material/test/renderers/MaterialTimeControl.test.tsx
@@ -377,7 +377,7 @@ describe('Material time control', () => {
     const input = wrapper.find('input').first();
     expect(input.props().value).toBe('02-13');
 
-    (input.getDOMNode() as HTMLInputElement).value = '12:01';
+    (input.getDOMNode() as HTMLInputElement).value = '12-01';
     input.simulate('change', input);
     expect(onChangeData.data.foo).toBe('1//12 am');
   });

